RUN3/AliAnalysisTaskAO2Dconverter.cxx

```diff
@@ -2031,17 +2045,19 @@ void AliAnalysisTaskAO2Dconverter::FillEventInTF()
         tracks.fRho1PtSnp = 0;
         tracks.fRho1PtTgl = 0;
         tracks.fTPCinnerP = NAN;
-        tracks.fFlags = 0;
+        tracks.fFlags = mlt->FreeClustersTracklet(itr, 0) && mlt->FreeClustersTracklet(itr, 1);
```

Comment:
The content of tracks.fFlags follows the enum TrackFlagsRun2Enum in DataTypes.h (O2). Please prepare also a comment change there, or even simply a second name with the same enum value (0x1).

Reply:
@jgrosseo thanks for the review. As you suggested, I added a new name specifically for tracklets in O2 (also in AliPhysics for completeness), here is the link to the PR: https://github.com/AliceO2Group/AliceO2/pull/13393